federatedscope/core/configs/cfg_hpo.py

```diff
@@ -81,5 +99,9 @@ def assert_hpo_cfg(cfg):
                                                        "federate.use_diff " \
                                                        "must be True."
 
+    assert cfg.hpo.fts.use == cfg.federate.use_diff, "Once FTS is " \
```

Comment:
Suppose I want to conduct HPO by FedEx with the difference of validation performances before and after local training as the feedback, then `fits.use` is false and `use_diff` is true. This assertion seems to be invoked.